Remove genomelake

haimasree opened the pull request:
genomelake removed as discussed. I have used your one_hot_encode_sequence and the extractors from genomelake and put them in a separate file for future reference. I changed test_github_permalink to include a model which does not have genomelake as dependency. I would get the following error otherwise - 
```
module genomelake not found
```
Is that okay? 

I am keeping this pr open - we can discuss on monday.

Avsecz commented on docs/templates/contributing/04_Writing_dataloader.py.md:
```diff
@@ -85,9 +85,9 @@ Here is an example dataloader that gets as input a [fasta](http://genetics.bwh.h
 from kipoi.metadata import GenomicRanges
+from kipoi.helper import FastaExtractor
 
 class SeqDataset(Dataset):
```
Could be replaced by kipoiseq.FastaStringExtractor followed by kipoiseq.transforms.one_hot_dna

Avsecz commented on example/models/extended_coda/dataloader.py:
```diff
@@ -4,7 +4,7 @@
 import pybedtools
 
-from genomelake.extractors import ArrayExtractor
+from kipoi.helper import ArrayExtractor
```
This test could be removed

Avsecz commented on example/models/non_bedinput_model/dataloader.py:
```diff
@@ -9,13 +9,11 @@
-
 from kipoi.data import Dataset
-
+from kipoi.helper import BaseExtractor, FastaExtractor, NUM_SEQ_CHARS, one_hot_encode_sequence
```
Could copy in BaseExtractor from genomelake here and use FastaStringExtractor from kipoiseq

Avsecz commented on example/models/pyt/dataloader.py:
```diff
@@ -7,7 +7,7 @@
 import pybedtools
 from pybedtools import BedTool
-from genomelake.extractors import FastaExtractor
+from kipoi.helper import FastaExtractor
```
Use FastaStringExtractor from kipoiseq

Avsecz commented on example/models/rbp/dataloader.py:
```diff
@@ -12,13 +12,14 @@
 from kipoi.metadata import GenomicRanges
 import linecache
 from kipoi.data import Dataset
+from kipoi.helper import BaseExtractor, FastaExtractor, NUM_SEQ_CHARS, one_hot_encode_sequence
```
See above

Avsecz commented on kipoi/helper.py:
```diff
@@ -0,0 +1,167 @@
+import six
```
Please remove this

haimasree commented:
I have addressed all the review and have moved one_hot_dna to https://github.com/kipoi/kipoiseq/tree/add-one-hot-encode. kipoiseq branch needs to be updated after one_hot_dna is added to kipoiseq's master branch
